esp/scm/ws_dfuXref.ecm

```diff
@@ -213,6 +213,9 @@ ESPrequest DFUXRefUnusedFilesRequest
     string ProcessCluster;
     bool CheckPackageMaps;
     [min_ver("1.01")] bool GetFileDetails(false);
+    [min_ver("1.02")] ESParray<string> ProcessClusterList;
```

Comment:
@richardkchapman You should actually increase the version to 1.03.. including changing below:

version("1.03"), default_client_version("1.03")